cfgov/unprocessed/css/organisms/secondary-navigation.less

```diff
@@ -0,0 +1,161 @@
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-secondary-navigation
+            .o-secondary-navigation_button
+            .o-secondary-navigation_list.o-secondary-navigation_list__parents
+              .o-secondary-navigation_item.o-secondary-navigation_item__parent
+    - name: Secondary nav mixed with cf-expandable
+      markup: |
+        <nav class="expandable
+                    o-secondary-navigation">
+            <div class="expandable_header">
+                <button class="expandable_target
+                               o-secondary-navigation_button">
+                    <span class="expandable_header-left">
+                        Nav Button
+                    </span>
+                    <span class="expandable_header-right">
+                        <span class="expandable_cue-open">
+                            <span class="cf-icon
+                                         cf-icon-down"></span>
+                        </span>
+                        <span class="expandable_cue-close">
+                            <span class="cf-icon
+                                         cf-icon-up"></span>
+                        </span>
+                    </span>
+                </button>
+            </div>
+            <div class="expandable_content">
+                <ul class="o-secondary-navigation_list
+                           o-secondary-navigation_list__parents">
+                    <li class="o-secondary-navigation_item
+                               o-secondary-navigation_item__parent">
+                        nav-link
+                    </li>
+                    <li class="o-secondary-navigation_item
+                               o-secondary-navigation_item__parent">
+                        nav-link
+                    </li>
+                    <li class="o-secondary-navigation_item
+                               o-secondary-navigation_item__parent">
+                        nav-link
+                    </li>
+                </ul>
+            </div>
+        </nav>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .expandable.o-secondary-navigation
+            .expandable_header
+              .expandable_target.o-secondary-navigation_button
+                .expandable_header-left
+                .expandable_header-right
+                  .expandable_cue-open
+                    .cf-icon.cf-icon-down
+                  .expandable_cue-close
+                    .cf-icon.cf-icon-up
+            .expandable_content
+              .o-secondary-navigation_list.o-secondary-navigation_list__parents
+                .o-secondary-navigation_item.o-secondary-navigation_item__parent
+  tags:
+    - cfgov-organisms
+*/
+
+.o-secondary-navigation {
+    .webfont-regular();
+
+    .respond-to-max(@bp-sm-max, {
+        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
```

Comment:
Are you saying the `content_sidebar` should do it? I can see that, though it'll require some tweaking because it has a `flush-all-on-small` modifier.